kitchensink/src/app/common/PageElements/FoundationElements/ColorRow/ColorRow.jsx

```diff
@@ -0,0 +1,167 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import ColorBox from '../ColorBox/ColorBox';
+import './colorRow.css';
+
+const ColorRow = (props) => (
+  <div styleName="colorRowContainer">
+    <table>
+      {
+        (props.showInfo && (props.showInfo === 'fullInfo' || props.showInfo === 'onlyTitles'))
+        && (
+          <thead>
+            <tr>
+              {
+                props.title.length > 0 && <th key="base-col-header" style={{ width: (props.rowHeader.width ? props.rowHeader.width : 'auto'), textAlign: 'left' }} styleName={classNames('title', { [props.rowSize]: true })}>{props.title}</th>
+              }
+              {
+                props.preColumns > 0
+                && <th colSpan={props.preColumns}>&nbsp;</th>
+              }
+              {
+                (Object.keys(props.rowHeader || {}).length > 0)
```

Comment:
> Why `|| {}`?
@justmejulian Because so you do not need to do so :

`props.rowHeader && props.rowHeader.length > 0 && Object.keys(props.rowHeader )`